tests/RA_Achievement_Tests.cpp

```diff
@@ -1,6 +1,6 @@
 #include "CppUnitTest.h"
 
-#include "RA_Achievement.h"
+#include "RA_AchievementSet.h"
```

Comment:
It's a cpp file anyway. To keep it consisted with typing out `AchievementSet::Type`.

Reply:
This shouldn't be necessary if you eliminate `IsCoreAchievement`, which might also solve the issue that was requiring the `Document` support for unit tests.

Reply:
Alright I'll try it out.